core/server/worker/src/main/java/alluxio/worker/grpc/AbstractWriteHandler.java

```diff
@@ -71,6 +84,7 @@
    */
   AbstractWriteHandler(StreamObserver<WriteResponse> responseObserver) {
     mResponseObserver = responseObserver;
+    mSerializingExecutor = new SerializingExecutor(GrpcExecutors.BLOCK_WRITER_EXECUTOR);
```

Comment:
the serializing executor only runs 1 task at a time, in order. Why does it need a large threadpool?

Reply:
It only serialize tasks within the executor for a single request. There can be multiple concurrent requests which requires more threads for processing messages concurrently.